cms/templates/settings.html

```diff
@@ -292,6 +292,26 @@ <h2 class="title-2">${_("Introducing Your Course")}</h2>
                 <span class="tip">${_("Information for prospective students")}</span>
               </header>
               <ol class="list-input">
+
+                % if about_page_editable:
+                <li class="field" id="field-course-title">
+                  <label for="course-title">${_("Course Title")}</label>
+                  <input type="text" id="course-title" data-display-name="${context_course.display_name}">
+                </li>
+                <li class="field" id="field-course-subtitle">
+                  <label for="course-subtitle">${_("Course Subtitle")}</label>
+                  <input type="text" id="course-subtitle">
+                </li>
+                <li class="field" id="field-course-duration">
+                  <label for="course-duration">${_("Course Duration")}</label>
+                  <input type="text" id="course-duration">
+                </li>
+                <li class="field text" id="field-course-description">
```

Comment:
Since all these fields are being treated as free-form text, shouldn't they all have the class "text"? Right now that is only present here on the Course Description field.